pyproject.toml

```diff
@@ -0,0 +1,73 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "dandisets-linkml-status"
+dynamic = ["version"]
+description = 'A tool for reporting the status of the translation of DANDI schemas from Pydantic definitions to LinkML definitions'
+readme = "README.md"
+requires-python = ">=3.9"
+license = "MIT"  # todo: Is this the correct license?
```

Comment:
@yarikoptic Can you help input the correct license info. The current info is generated by Hatch mostly. I am perfectly fine the direct the license to DANDI. I just don't know the proper way to put it.